playwright-e2e/playwright.config.ts

```diff
@@ -36,7 +36,7 @@ const testConfig: PlaywrightTestConfig = {
       scale: 'css',
       // Account for minor difference in text rendering and resolution between headless and headed mode
       threshold: 1,
-      maxDiffPixelRatio: 0.5
+      maxDiffPixelRatio: 1
```

Comment:
this change is specifically for the `toHaveScreenshot()` method since that's what's failing

Reply:
ok that didn't seem to fix it just yet (ran feature branch on circleci)